docs/changelog.rst

```diff
@@ -4,6 +4,13 @@
 Changelog
 =========
 
+- :release:`7.2.0 <28th June 2022>` 93
```

Comment:
I don’t think this needs to include `93`. It should already be included by default as a feature in a feature release.